main.tf

```
@@ -146,7 +146,7 @@ resource "aws_launch_configuration" "kong" {
   associate_public_ip_address = false
   enable_monitoring           = true
   placement_tenancy           = "default"
-  user_data                   = data.template_cloudinit_config.cloud_init.rendered
+  user_data                   = var.custom_user_data == "false" ? data.template_cloudinit_config.cloud_init.rendered : var.user_data
```

in line with comments on variables.tf
```suggestion
  user_data                   = var.user_data == null ? data.template_cloudinit_config.cloud_init.rendered : var.user_data
```

---

Latest commit has applied suggestion